.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
```

Comment:
Please add license header to this file, too.

Reply:
Addressed this in the latest commit.